qutip/qip/operations/gates.py

```diff
@@ -44,9 +44,9 @@
 from qutip.states import fock_dm
 
 
-__all__ = ['x_gate', 'y_gate', 'z_gate', 's_gate', 't_gate',
-           'rx', 'ry', 'rz', 'sqrtnot', 'snot', 'phasegate', 'qrot',
-           'cphase', 'cnot',
+__all__ = ['rx', 'ry', 'rz', 'sqrtnot', 'snot', 'phasegate', 'qrot',
+           'x_gate', 'y_gate', 'z_gate', 'cy_gate', 'cz_gate', 's_gate',
```

Comment:
***CX_GATE*** aka ***CNOT*** was already implemented

Reply:
Aha, yes you are right.

Reply:
It might be good to put in a `cx_gate` just for consistency.  You can just define the gate through `cx_gate = cnot` (or whatever it is) and add the extra line to `circuit.py`.

People are more likely to access it through `cnot`, but it's kind of weird to define a `cy_gate` and a `cz_gate` but have `cx_gate` only accessible by a completely different name.  It just seems like an unnecessary complication to remember that you can't have it that way.

The duplication is similar to how we have both `qeye` and `identity` to construct an identity operator.

Reply:
Aha... sorry I missed this comment.